src/scenarios/shared/runner.py

```diff
@@ -62,11 +70,39 @@ def run(self):
             startup.runtests(**self.traits._asdict(),
                              scenariotypename=const.SCENARIO_NAMES[const.STARTUP],
                              apptorun=publishedexe(self.traits.exename))
-        # if testtype == 'sdk' and self.traits.sdk:
-        #     print("sdk")
-        #     startup = StartupWrapper()
-        #     startup.runtests(**self.traits._asdict(),
-        #         scenariotypename='Build No Changes')
-        #     # fix some other traits
-        #     startup.runtests(**self.traits._asdict(),
-        #         scenariotypename='Rebuild')
\ No newline at end of file
+        elif self.testtype == const.SDK:
+            startup = StartupWrapper()
+            envlistbuild = 'DOTNET_MULTILEVEL_LOOKUP=0'
+            envlistcleanbuild= envlistbuild+';'+'MSBUILDDISABLENODEREUSE=1'
+            # clean build
+            startup.runtests(scenarioname=self.traits.scenarioname,
+                             exename=self.traits.exename,
+                             guiapp=self.traits.guiapp,
+                             startupmetric=const.STARTUP_PROCESSTIME,
+                             appargs='build',
+                             timeout=self.traits.timeout,
+                             warmup='false',
```

Comment:
why should we not warm up the clean build case? It's still going to run many iterations so logically the first will be "warmup" anyway.

Reply:
true. The first-ever iteration could be much slower and a warmup iteration could make all the iterations consistent.